src/main/java/de/unistuttgart/overworldbackend/service/PlayerTaskStatisticService.java

```diff
@@ -181,7 +181,8 @@ private long calculateKnowledge(final long score, final long highscore) {
       (double) Math.max(0, score - highscore) /
       100 +
       MAX_KNOWLEDGE *
-      Math.max(0, score - Math.max(0, score - highscore)) *
+      Math.max(0, score - Math.max(0, score - highscore)) /
+      100 *
```

Comment:
Somewhat unrelated, but because of this formatting, I have a really hard time understanding what is done when.
Please add parentheses.

Reply:
That is not what I meant.
I meant the explicit arithmetic order.
I find it surprisingly difficult to understand the order of when what is done with this formatting.